lib_bagl/src/bagl_font_open_sans_extrabold_11px.inc

```diff
@@ -185,7 +183,8 @@ const unsigned char bitmapOPEN_SANS_EXTRABOLD_11PX[] = {
 //ascii 0x0078
   0x77, 0x1B, 0x87, 0x63, 0xBB, 0x03,
 //ascii 0x0079
-  0x63, 0x9B, 0x8D, 0xC6, 0xE1, 0x30, 0x18, 0x06,
+  0x63, 0x36, 0x36, 0x34, 0x1C, 0x1C, 0x1C, 0x0C,
+  0x0E,
```

Comment:
Is this intended? If I understand correctly this will change the apperance of the "y" letter?

Reply:
I can't generate .json files without generating corresponding .inc files at the same time, and as you noticed, some changes have been made on some letters.
(It's supposed to be an improvement, not a regression)

Generating an app without using those .inc files is not mandatory, but it will result in OCR failing to recognize the couple of letters which are different.

An other solution would be to write a **inc2json.py** script, but this is a lot of work for just _one shot_...

Reply:
As I told you on Slack, I was able to use the pictures corresponding to API_LEVEL_5 tag and the generated .inc files was identical => this PR will now just include the .json files.

Reply:
Did you pushed it? I still see changes in `.inc` files.